apps/microtvm/arduino/template_project/tests/test_arduino_microtvm_api_server.py

```diff
@@ -64,6 +65,7 @@ def test_find_modified_include_path(self, mock_pathlib_path):
 
     BOARD_CONNECTED_OUTPUT = bytes(
         "Port         Type              Board Name          FQBN                        Core             \n"
+        "/dev/ttyACM1 Serial Port (USB) Wrong Arduino       arduino:mbed_nano:nano33    arduino:mbed_nano\n"
```

Comment:
can you move this down one line, so that if the logic was naive and just looking for `"nano33" in FQBN`, it would find /dev/ttyACM0 first?